addons/account/i18n/es.po

```diff
@@ -9786,7 +9786,7 @@ msgstr "No"
 #. module: account
 #: model_terms:ir.ui.view,arch_db:account.view_account_payment_search
 msgid "No Bank Matching"
-msgstr ""
+msgstr "No conciliado con banco"
```

Comment:
@pablohmontenegro This one should be done through Transifex, I am afraid.

Reply:
Thanks! I made the change suggested @jco-odoo